imports/ui/Body.js

```diff
@@ -1,40 +1,48 @@
 import { Template } from 'meteor/templating';
-
 import { Players } from '../api/Players.js';
+import { Constants } from '../api/Constants.js';
 
 import './About.html';
-import './Body.html';
 import './Home.html';
+import './Body.html';
```

Comment:
Why is this reordered?  I'd prefer if imports were in alphabetical order (I know they weren't before but I also tried to group similar imports)